SwanContents/swancontents/__init__.py

```diff
@@ -1,36 +1,22 @@
 from ._version import __version__ 
-from .filemanager import *
-from .handlers import *
+# from .filemanager import *
```

Comment:
Can we just remove these two lines?

Reply:
This is already removed in a later commit.

Reply:
But I also cleaned this in a soon to be pushed commit